mlos_core/mlos_core/spaces/adapters/llamatune.py

```diff
@@ -100,7 +100,10 @@ def inverse_transform(self, configurations: pd.DataFrame) -> pd.DataFrame:
             # respective high-dim point; this way we can retrieve the low-dim point, from its high-dim counterpart.
             if target_config is None:
                 # Inherently it is not supported to register points, which were not suggested by the optimizer.
-                if not self._use_approximate_reverse_mapping:
+                if configuration == self.orig_parameter_space.get_default_configuration():
+                    # Default configuration should always be registerable.
```

Comment:
This addresses the issue of us proposing the default first without really consulting the optimizer.
One could potentially argue that we should set `use_approximate_reverse_mapping` to `True` by default, but I think this was a smaller scoped change and sane to do for now.

Reply:
@kkanellis in case he wants to comment further